python/triton/compiler/compiler.py

```diff
@@ -304,7 +304,8 @@ def compile(src, target=None, options=None):
     # This is needed to safely finalize threads pool inside context: if current process forks before
     # python GC deletes context object, thread pool in child process will be invalid, which could
     # lead to child crash or hang.
-    context.disable_multithreading()
+    if not os.environ.get("TRITON_ENABLE_ASAN", "0") == "1":
```

Comment:
Why should we enable multi-threading?

Reply:
Disabling multi-threading causes ASAN to create a deadlock. We're still hunting down why.